nessie/src/test/java/org/apache/iceberg/nessie/TestNessieCatalog.java

```diff
@@ -160,4 +162,15 @@ protected boolean supportsServerSideRetry() {
   public void testConcurrentCreateTransaction() {
     super.testConcurrentCreateTransaction();
   }
+
+  @Test
+  public void testWarehouseLocationWithTrailingSlash() {
+    Assertions.assertThat(catalog.defaultWarehouseLocation(TABLE))
+        .startsWith(
+            LocationUtil.stripTrailingSlash(temp.toUri().toString())
```

Comment:
This testcase was failing without the fix as `temp.toUri()` was having trailing slash which was leading to double slash in the path.